### DISCUSSION TITLE
Dynamic properties are deprecated now.

### DISCUSSION THREAD

**Discussion by marcinkleczek** [chevere/xr#82] [Q&A]
Creation of dynamic property Chevere\Xr\VarDump\Output\XrVarDumpHtmlOutput::$caller is deprecated

**Comment by rodber** [comment]
https://github.com/chevere/xr/blob/1.0/src/VarDump/Output/XrVarDumpHtmlOutput.php
https://github.com/chevere/var-dump/blob/5414f67e99affc413df2ad42f340eaebb8616b87/src/Outputs/Output.php#L29

```sh
❯ composer show chevere/var-dump
name     : chevere/var-dump
descrip. : A multi-purpose colorful modern alternative to var_dump
keywords : chevere, dd, var_dump
versions : * 0.9.0
```
